starlette/routing.py

```diff
@@ -469,8 +470,14 @@ def decorator(func: typing.Callable) -> typing.Callable:
     async def __call__(self, scope: Scope, receive: Receive, send: Send) -> None:
         message = await receive()
         assert message["type"] == "lifespan.startup"
-        await self.startup()
-        await send({"type": "lifespan.startup.complete"})
+        try:
+            await self.startup()
+        except BaseException as exc:
+            msg = traceback.format_exc()
+            await send({"type": "lifespan.startup.failed", "message": msg})
+            raise exc from None
```

Comment:
Why is `from None` here used?

Reply:
Hm, personal code style preference. I’ll remove it. :-)

Reply:
This and your other comment should be fixed now.